justega247/favorite-things
# #167373357: Implement the basic setup for the frontend

#### What does this PR do?
- This PR is for the initial setup for the frontend part of the application
#### Description of Task to be completed?
- create the client directory to house all frontend related code
- implement the basic setup for the frontend implementation
#### How should this be manually tested?
- Clone the project
- Navigate to the client directory
- Run **npm start**
- On your browser, navigate to **http://127.0.0.1:3000/** when the application starts
- You should see **hi hello vue** displayed on the page
#### What are the relevant pivotal tracker stories?
[#167373357](https://www.pivotaltracker.com/story/show/167373357)
